src/reuse/download.py

```diff
@@ -31,6 +34,8 @@
     "https://raw.githubusercontent.com/spdx/license-list-data/master/text/"
 )
 
+REF_RE = re.compile("LicenseRef-[a-zA-Z0-9-.]+$")
```

Comment:
Just a minor issue, but should this be defined here or rather in `_licenses.py` to keep everything in the same central place?

Reply:
I don't have a strong opinion, but it might make sense to put it there. In the rest of the code base we don't use a regex for LicenseRef yet but just `.startswith("LicenseRef-")`, but it may well be that this will change for some reason someday.

Reply:
@floriansnow can we merge this as is or do you want this changed?

Reply:
@nicorikken I can live with both, but if we merge it as is, let's create an issue to clean this up a bit at a later date, together with the instances Max mentioned.

Reply:
No need to create an issue, I moved the regexp and updated the `LicenseRef-` check occurrences.